modules/driver-jdbc/src/test/kotlin/datamaintain/db/driver/jdbc/JdbcDriverTest.kt

```diff
@@ -138,14 +141,16 @@ internal class JdbcDriverTest {
                 executionStatus = ExecutionStatus.OK,
                 action = ScriptAction.OVERRIDE_EXECUTED)
 
+        val lightScript3 = LightExecutedScript(script3.name, script3.checksum, script3.identifier)
+
         // When
         jdbcDatamaintainDriver.overrideScript(script3)
 
         // Then
         expectThat(jdbcDatamaintainDriver.listExecutedScripts().toList())
                 .containsExactlyInAnyOrder(
-                        script3,
-                        script2
+                        lightScript3,
+                        lightScript2
```

Comment:
No it does not work without explicit type.

Reply:
And casting is not enough.

Reply:
Then you could add a function extension to ```ExecutedScript```, ```toLightExecutedScript()```. I'm sure it will help for other tests because ```MongoDriverTest``` should also check stuff like that.

Reply:
Already done